.scalafmt.conf

```diff
@@ -4,6 +4,7 @@ continuationIndent.defnSite = 2
 assumeStandardLibraryStripMargin = true
 includeCurlyBraceInSelectChains = false
 trailingCommas = always
+docstrings.style = keep
```

Comment:
Formatting docstrings seems to cause two problems:

1. if we format the code used for metarpheus snapshot tests, the tests fail because the param description (in docstring `@param` lines) get broken up in separate lines, which metarpheus doesn't support
2. indenting in code blocks inside docstrings is messed up, for example in https://github.com/buildo/retro/blob/cc43736/enumero/core/src/main/scala/CaseEnum.scala#L13 reformatting would remove the indent

1 could be solved by not reformatting the fixtures code (or improving metarpheus), but considering also 2 I thought maybe it makes sense to disable docstring formatting altogether. What do you think?

Reply:
Not a huge deal, let's leave it off